test/e2e/tls_test.go

```diff
@@ -55,20 +67,8 @@ func TestPeerTLS(t *testing.T) {
 		}
 	}()
 
-	members, err := e2eutil.WaitUntilSizeReached(t, f.KubeClient, 3, 60*time.Second, c)
+	_, err = e2eutil.WaitUntilSizeReached(t, f.KubeClient, 3, 60*time.Second, c)
```

Comment:
shall we try to use the client key/certs to talk to the secure etcd as a client to verify client TLS works here? (i understand that operator itself is already a client, but i would like to see our test being explicit)